### PR TITLE
Fix(InvestigationTable): Close dropdown out of cell bug #1082

### PR DESCRIPTION
**Pull request by doronlnd84**

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 1]
```diff
@@ -401,8 +411,13 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
     return (
-        <>
-            <Grid className={classes.title} container alignItems='center' justify='space-between'>
+        <div tabIndex={0}
+            onClick={(e: any) => closeDropdowns()}
+            onKeyDown={(e: any) =>
+                e.key === 'Escape' && closeDropdowns()}
+        >
+            <Grid className={classes.title} container alignItems='center' justify='space-between'
+            >
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:420]: move it to line 419

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 2]
```diff
@@ -401,8 +411,13 @@ const InvestigationTable: React.FC = (): JSX.Element => {
     }, [tableRows, unassignedInvestigationsCount]);
 
     return (
-        <>
-            <Grid className={classes.title} container alignItems='center' justify='space-between'>
+        <div tabIndex={0}
+            onClick={(e: any) => closeDropdowns()}
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:415]: change type

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 3]
```diff
@@ -401,8 +411,13 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
+            onKeyDown={(e: any) =>
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:416]: same

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 4]
```diff
@@ -457,7 +472,8 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                         </Button>
                     </Box>
                 </div>
-                <Grid container justify="flex-end" className={classes.filterTableRow}>
+                <Grid container justify="flex-end" className={classes.filterTableRow}
+                >
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:476]: same

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 5]
```diff
@@ -486,13 +502,14 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                     </Collapse>
                 </Grid>
                 <TableContainer ref={tableContainerRef} component={Paper} className={classes.tableContainer}>
-                    <Table aria-label='simple table' stickyHeader id='LandingPageTable'>
+                    <Table aria-label='simple table' stickyHeader id='LandingPageTable'
+                    >
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:506]: same

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 6]
```diff
@@ -532,6 +549,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                                                     {
                                                         getTableCell(key, indexedRow)
                                                     }
+
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:552]: delete row

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 7]
```diff
@@ -247,13 +254,16 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                 }
             case TableHeadersNames.county:
                 if (selectedRow === indexedRow.epidemiologyNumber && countyAutoCompleteClicked) {
+
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:257]: delete row

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx (hunk 2 above)
Comment (daniel7764) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:415]: change e to event

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx (hunk 3 above)
Comment (daniel7764) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:416]: same

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 8]
```diff
@@ -401,7 +410,11 @@ const InvestigationTable: React.FC = (): JSX.Element => {
     }, [tableRows, unassignedInvestigationsCount]);
 
     return (
-        <>
+        <div tabIndex={0}
+            onClick={() => closeDropdowns()}
```
Comment (daniel7764) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:414]: i think you can do it like this: 
onClick={closeDropDowns}, and not defining a function

**Review by ItayBenMoshe — DISMISSED** [review]

**Review by daniel7764 — APPROVED** [review]

**Review by daniel7764 — APPROVED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Pull request merged**